pep/app/services/current-user.ts

```diff
@@ -215,7 +215,7 @@ export default class CurrentUserService extends Service {
      * @param {Document} document
      * @memberof CurrentUserService
      */
-    addDocument(key: PreferenceDocumentsKey, documentId: string) {
+    addPreferenceDocument(key: PreferenceDocumentsKey, documentId: string) {
         const prefs = this.loadLocalStoragePrefs();
```

Comment:
these should still not be using `this.loadLocalStoragePrefs();`, it should just use `this.preferences`. same for the other document-related methods below.